schema/note.mod

```diff
@@ -428,13 +428,28 @@
 
 	The number attribute is rarely needed to disambiguate ties,
 	since note pitches will usually suffice. The attribute is
-	implied rather than defaulting to 1 as with most elements.
-	It is available for use in more complex tied notation
-	situations.
+	implied rather than defaulting to 1 as with most elements. It is
+	available for use in more complex tied notation situations.
+
+	Ties that join two notes of the same pitch together should be

```

Comment:
Similar suggestion to the XSD regarding rewording the type="start" constructs to be more similar in style to other MusicXML documentation.